### ISSUE TITLE
attributes loss 계산할 때

### ISSUE THREAD

**Issue by namghiwook** [jeong-tae/DeepFashion-Tensorflow#1]
제가 잘은 모르지만.. 
논문을 보면 아래 내용이 있던데

Third, a weighted cross-entropy loss is utilized to predict attributes .....

attributes loss 계산할 때 tf.nn.weighted_cross_entropy_with_logits 사용해야하는거 아닌가요?

**Comment by jeong-tae** [comment]
네. 맞습니다. 제가 일단 코딩은 attribute의 weight를 생각하지 않고 만들었습니다.
attribute의 positive 샘플과 negative 샘플 수를 구하여서 weight를 계산하도록 곧 반영할 예정입니다.
